yaksh/test_views.py

```diff
@@ -1758,6 +1760,141 @@ def tearDown(self):
         self.student.delete()
         self.user1_course.delete()
 
+
+    def test_upload_users_with_correct_csv(self):
+        # Given
+        self.client.login(
+            username=self.user1.username,
+            password=self.user1_plaintext_pass
+        )
+        csv_file_path = os.path.join(settings.FIXTURE_DIRS, "users_correct.csv")
```

Comment:
please avoid using FIXTURE_DIRS as the PR #379 deprecates this value in ```settings.py``` and movese it to ```models.py``` as FIXTURES_DIR_PATH
Please see https://github.com/ankitjavalkar/online_test/blob/3ff5eeeb1066d579213c9ca7b62c1e39d774166c/yaksh/models.py#L85

Reply:
Yes, the PR #379 is been merged so will fix it!